api/nakadi-event-bus-api.yaml

```diff
@@ -392,6 +394,46 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
 
+  /event-types/{name}/effective_schema:
```

Comment:
1) For me this endpoint looks not completely RESTful. Were there any alternatives for endpoint URL?
2) Why do you use underscore here? I think for other places we use dashes (e.g. `event-types`)